.github/workflows/check-versions.yaml

```diff
@@ -32,6 +32,30 @@ jobs:
           ./.github/scripts/update-versions.sh
           git diff
 
+      - name: Setup go
+        uses: actions/setup-go@v4.0.0
```

Comment:
Why do we need Go in this pipeline? Doesn't seem like we need to compile any Go here.

Reply:
left over from yq I think. Will remove.

Reply:
oh. yq is still used.